test/e2e/service.go

```diff
@@ -966,6 +968,89 @@ var _ = framework.KubeDescribe("Services", func() {
 			framework.Failf("expected un-ready endpoint for Service %v within %v, stdout: %v", t.name, kubeProxyLagTimeout, stdout)
 		}
 	})
+
+	It("should be able to create services of type LoadBalancer and externalTraffic=localOnly [Slow]", func() {
```

Comment:
Should probably add a [Feature:externalTrafficLocalOnly] tag so this doesn't get picked up by regular serial tests.